lms/views/dashboard/views.py

```diff
@@ -128,8 +126,8 @@ def _set_lti_user_cookie(self, response):
             value=auth_token,
             secure=not self.request.registry.settings["dev"],
             httponly=True,
-            # Scope the cookie to all the org endpoints
-            path=f"/dashboard/organizations/{lti_user.application_instance.organization.public_id}",
+            # Scope the cookie to all dashboard views
```

Comment:
Auth is no longer scoped to one organization as you might seeing information for more than one.

This doesn't handle the case where you have more that one LTI idnenitty, eg you use Canvas and Moodle.

We'll leave that out of scope for now.